bin/test_topology

```diff
@@ -18,17 +18,32 @@ fi
 
 function test_faucetizer {
     test_case=$1
-    fc_file=faucet.yaml
+    fc_file=faucet_structural.yaml
     state_file=devices_$test_case.yaml
-    output_file=faucet_$test_case.yaml
     sample_file=faucet_$test_case.yaml
+    output_file=faucet_faucet.yaml
 
     $PYTHON -m forch.faucetizer -s $state_file -c $fc_file -o $output_file
     echo Comparing $OUT_DIR/$output_file and $FORCH_CONFIG_DIR/$sample_file
     (diff $OUT_DIR/$output_file $FORCH_CONFIG_DIR/$sample_file && echo 0 || echo 1) >> $TEST_RESULTS
 }
 
+function test_stack {
+    desc=stack-$1
+    echo Starting $desc test... | tee -a $TEST_RESULTS
+    wget localhost:8001 -O- 2>/dev/null | grep 'nz-kiwi-t2sw1' | grep -oE '9a:02:57:1e:8f:01' | tee -a $TEST_RESULTS
```

Comment:
what is this endpoint (8001)?  Is this the faucet varz?

Reply:
Yes, was going to make it a parameter but forgot..